server/services/dashboard/dashboard.go

```diff
@@ -88,6 +98,15 @@ func (t *dashboard) getDashboardGrid() ([][]*schema.DashboardElementView, error)
 				if err != nil {
 					return nil, err
 				}
+			case schema.DashboardElementTypeTCBuilds:
+				tcBuildInfo, err := t.sideStats.TCBuildInfo()
+				if err != nil {
+					return nil, err
+				}
+
+				elementView.BuildStatuses = tcBuildInfo.BuildStatuses
+				elementView.LastChangesAuthor = types.NewNullString(tcBuildInfo.LastChangesAuthor)
```

Comment:
i think you should provide full structs to client, and it will take info it needs